### PR TITLE
Adapter to Connect to the RERUM Sandbox

### PR DESCRIPTION
**Pull request by thehabes**

### CODE REVIEW

**Review by cubap — APPROVED** [review]

File: demo/src/index.js [hunk 1]
```diff
@@ -1,14 +1,12 @@
-
 import mirador from 'mirador/dist/es/src/index';
 import annotationPlugins from '../../src';
 import LocalStorageAdapter from '../../src/LocalStorageAdapter';
-import AnnototAdapter from '../../src/AnnototAdapter';
+import RerumAdapter from '../../src/RerumAdapter';
```
Comment (cubap) [inline demo/src/index.js:5]: maybe have this commented out or at least don't replace the AnnototAdapter

File: demo/src/index.js [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 
-const endpointUrl = 'http://127.0.0.1:3000/annotations';
 const config = {
   annotation: {
     adapter: (canvasId) => new LocalStorageAdapter(`localStorage://?canvasId=${canvasId}`),
-    // adapter: (canvasId) => new AnnototAdapter(canvasId, endpointUrl),
+    // adapter: (canvasId) => new RerumAdapter(canvasId), // put Annotations online with RERUM
```
Comment (cubap) [inline demo/src/index.js]: recommend including optional params in the comment for completeness:

```
// adapter: (canvasId) => new RerumAdapter(canvasId, endpointUri?, creator?), // put Annotations online with RERUM
```

**Review by cubap — APPROVED** [review]

File: demo/src/index.js (hunk 2 above)
Comment (thehabes) [reply demo/src/index.js]: argh dang, I removed them all when I had it uncommented for the functionality.  That's a fine idea to put the params into the comment, that won't hurt anything.

File: demo/src/index.js (hunk 1 above)
Comment (thehabes) [reply demo/src/index.js:5]: Ok ok, I wasn't sure how completely we were removing the AnnototAdapter

**Pull request merged**